### DISCUSSION TITLE
Hide intermediate commits created by `git rebase -i` from `git sl`

### DISCUSSION THREAD

**Discussion by vgrigoriu** [arxanas/git-branchless#1272] [Q&A] [answered]
After I run `git rebase -i` and set all commits except the first one as fixups, I see in the output of `git sl` all the intermediate commits created during rebase, with messages like `# This is a combination of N commits. # This is the 1st commit message:` etc. Is there a way to hide them automatically?

**Comment by claytonrcarter** [comment, marked as answer]
> Is there a way to hide them automatically?

Not at present, but there is work in progress at https://github.com/arxanas/git-branchless/pull/1098, and further discussion at https://github.com/arxanas/git-branchless/issues/1088

The current workaround, as noted in #1088, is running something like `git hide 'stack() & message("This is a combination")'` after your rebase. If all of your work is on a branch, you may also have luck with something like `git hide 'stack() - main()..branches()'`, but beware that that will also hide any "branchless" commits you've created.
